src/handler/http/models/alerts/requests.rs

```diff
@@ -38,55 +38,33 @@ pub struct UpdateAlertRequestBody(pub Alert);
 #[derive(Debug, Deserialize, utoipa::IntoParams)]
 #[into_params(style = Form, parameter_in = Query)]
 #[serde(rename_all = "snake_case")]
+#[into_params(rename_all = "snake_case")]
 pub struct ListAlertsQuery {
     /// Optional folder ID filter parameter.
     pub folder: Option<String>,
 
-    #[serde(flatten)]
-    pub stream: Option<ListAlertsQueryStreamParams>,
-
     /// Optional owner user filter parameter.
     pub owner: Option<String>,
 
     /// Optional enabled filter parameter.
     pub enabled: Option<bool>,
 
-    /// The optional page size and page index of results to retrieve.
-    #[serde(flatten)]
-    pub page: Option<ListAlertsQueryPageParams>,
-}
```

Comment:
Unfortunately Utoipa doesn't seem like it's able to flatten nested structs in the generated Open API docs, so I'm replacing these nested structs with their child fields.